alphastats/loader/BaseLoader.py

```diff
@@ -74,8 +81,8 @@ def _check_if_file_exists(self, file):
 
     def _add_contamination_column(self):
         #  load df with potential contamination from fasta file
-        contaminations_path = pkg_resources.resource_filename(
-            __name__, "../data/contaminations.txt"
+        contaminations_path = (
+            importlib_resources.files(__package__) / "../data/contaminations.txt"
```

Comment:
why switch from __name__ to __package__?

Reply:
didn't work for name